.phpstan.neon

```diff
@@ -0,0 +1,5 @@
+parameters:
+    excludes_analyse:
+        - %rootDir%/vendor/*
+    ignoreErrors:
+        - '#Result of method React\\Socket\\ServerInterface::close\(\) \(void\) is used#'
```

Comment:
Why is this ignored?

Reply:
Because of usage "@" https://github.com/php-pm/php-pm/blob/master/ProcessManager.php#L238